test/prepare-test-environment.sh

```diff
@@ -5,11 +5,11 @@
 export GOOGLE_PROJECT='test-terraform-project-01'
 export TF_VAR_shared_vpc_host_google_project="test-gcp-project-01-274314"
 
-if ls ../gcp-gke/compute.json; then
+if ls compute.json; then
   export GOOGLE_CREDENTIALS=$(cat ../gcp-gke/compute.json)
```

Comment:
I think this way doesn’t work … because when you export and exit shell … value will be disappeared…

it need export when you run with other command in same script

Reply:
It will work if you run with `source` but this script can be install required software